src/lab7.py

```diff
@@ -57,550 +57,618 @@ def request(url):
 
     return headers, body
 
+def relative_url(url, current):
```

Comment:
absolute_url

Reply:
You're right. Perhaps `resolve_url`? This is going to need a PR because this function was introduced in the last chapter.